modules/data.atmosphere/R/download.CRUNCEP_Global.R

```diff
@@ -79,6 +99,14 @@ download.CRUNCEP <- function(outfolder, start_date, end_date, site_id, lat.in, l
       # This throws an error if file not found
       dap <- ncdf4::nc_open(dap_file)
 
+      # confirm that timestamps match
+      stopifnot(dap$dim$time$len == ntime)
```

Comment:
Would prefer if you use PEcAn's logger functions so that we can ensure that this stop is properly caught and written out to the log file.